pkg/arvo/lib/hood/drum.hoon

```diff
@@ -171,6 +175,11 @@
   =<  se-abet  =<  se-view
   (se-text "[{<src.hid>}, driving {<our.hid>}]")
 ::
+++  poke-safemode                                     ::
+  |=  saf/?
+  ^-  (quip move part)
+  [~ +<+.^$(ray (deft-apes our.hid saf))]
```

Comment:
Does this not need a call to `+se-abet`?

Reply:
Actually, it explicitly needs to *not* call `+se-abet`, or else talk crashes.  `+se-abet` starts any unstarted apps in `ray`.  I'm unsure the exact dependencies, but it seems that the peer to drum needs to be the first message that drum abets for everything to work right.